pkg/agent/baker.go

```diff
@@ -1424,6 +1424,7 @@ oom_score = -999{{if HasDataDir }}
 root = "{{GetDataDir}}"{{- end}}
 [plugins."io.containerd.grpc.v1.cri"]
   sandbox_image = "{{GetPodInfraContainerSpec}}"
+  enable_cdi = true
```

Comment:
did we also want to make the same change for scriptless right now? if so we'll need to modify the corresponding template here: https://github.com/Azure/AgentBaker/blob/master/aks-node-controller/parser/templates/containerd.toml.gtpl

Reply:
I also wonder if it would be worth adding validation for this setting in agentbaker e2e's